spec/components/summary_data_component_spec.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe SummaryDataComponent, type: :component do
+  pending "add some examples to (or delete) #{__FILE__}"
+
+  # it "renders something useful" do
```

Comment:
@jcoyne this needs cleanup of comments as well.

Reply:
done.